hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -43,26 +41,26 @@ Check that the output `0x3020000a` is shown. If it is not, then the OTP bit has
 
-This is the same [procedure](../../../installation/installing-images) as for SD cards.
+The [procedure](../../../installation/installing-images) is the same as for SD cards - simply image the USB storage device with the operating system image.
 
-After preparing the storage device, connect the drive to the Raspberry Pi and power up the Pi (be aware of the extra USB power usage from the external drive).
+After preparing the storage device, connect the drive to the Raspberry Pi and power up the Pi, being aware of the extra USB power usage from the external drive.
 After five to ten seconds, the Raspberry Pi should begin booting and show the rainbow splash screen on an attached display. Make sure that you do not have an SD card inserted in the Pi, since if you do, it will boot from that first.
 
-<a name="pi4"></a>
-## Raspberry Pi 4
+## Known issues (not Pi 4)
 
-USB mass storage support is currently undergoing beta testing.  Once beta test is complete the release version will enable USB mass storage boot by default. The `program_usb_boot_mode` option is not required on Pi 4.
+- The default timeout for checking bootable USB devices is 2 seconds. Some flash drives and hard disks power up too slowly. It is possible to extend this timeout to five seconds (add a new file `timeout` to the SD card), but note that some devices take even longer to respond.
+- Some flash drives have a very specific protocol requirement that is not handled by the bootcode and may thus be incompatible.
+- Lack of power can be an issue so it is recommended to use a powered USB hub, particularly if you are attaching more than one storage device to the Raspberry Pi. If your device has its own power supply, then use that.
 
-If you would like to try the beta test then please read the instructions on the [Pi 4 Bootloader Configuration](../bcm2711_bootloader_config.md).
+<a name="pi4"></a>
+## Raspberry Pi 4
 
-## Known issues
+USB mass storage boot for the Pi 4 is currently undergoing beta testing.  Once beta test is complete, USB mass storage boot will be enabled by default. The `program_usb_boot_mode` option is not required on Pi 4.
 
-- The default timeout for checking bootable USB devices is 2 seconds. Some flash drives and rotational harddrives power up too slowly. It’s possible to extend this timeout to five seconds (add a new file `timeout` to the SD card), but there are devices that fail to respond within this period as well.
-- Some flash drives have a very specific protocol requirement that is not handled by the bootcode and may thus be incompatible.
-- Lack of power can be an issue, so it is recommended to use a powered USB hub, particularly if you are attaching more than one storage device to the Raspberry Pi. If your device has its own power supply, then use that.
+If you would like to try the beta test, see the instructions on the [Pi 4 Bootloader Configuration](../bcm2711_bootloader_config.md).
```

Comment:
Should "see the instructions on the Pi 4 Bootloader Configuration." be expanded to ""see the instructions on the Pi 4 Bootloader Configuration page." ? :shrug: